catalogue_graph/terraform/state_machine_concepts_pipeline.tf

```diff
@@ -0,0 +1,60 @@
+resource "aws_sfn_state_machine" "concepts_pipeline" {
+  name     = "concepts-pipeline"
+  role_arn = aws_iam_role.state_machine_execution_role.arn
+
+  definition = jsonencode({
+    Comment       = "Build the catalogue graph and ingest concepts into ES",
+    QueryLanguage = "JSONata"
+    StartAt       = "Extractors"
+    States = {
+      "Extractors" = {
+        Type = "Map",
+        ItemProcessor = {
+          StartAt = "Extract nodes and edges from source",
+          States = {
+            "Extract nodes and edges from source" = {
```

Comment:
I think the extractors also need to run in a particular order so this might not work as expected.